addons/mail/static/src/views/fields/mail_server_configurator_selection/mail_server_configurator_selection.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates>
+    <div t-name="mail.MailServerConfiguratorSelection" class="d-flex flex-row">
+        <t t-if="readonly">
+            <span t-esc="string" t-att-raw-value="value" />
+        </t>
+        <t t-else="">
+            <select
+                class="o_input pe-3 me-3"
+                t-att-class="{'w-auto': value !== 'default'}"
+                t-on-change="(event) => this.onOptionChange(event.target.value)">
+                <option
+                    t-foreach="options" t-as="option" t-key="option[0]"
+                    class="text-black"
+                    t-att-selected="option[0] === value"
+                    t-att-value="option[0]"
+                    t-out="option[1]"
+                />
+            </select>
+            <button t-if="value != 'default' and isServerConfigured"
```

Comment:
This part should technically be a separate widget, configuring the mail server and testing the connections are 2 separate actions.

Can't we use a good old action button to test the connection actually? I feel like we already do that somewhere else. 
#lesscustomJSMorestandardcodeplease